scotty

```diff
@@ -202,9 +202,11 @@ def install(args):
 
     run_commands(install_command)
 
-    print_color("\nInstalling and activating virtual env")
-    create_venv()
-    start_venv()
+
+    if not is_ras_pi:
+        print_color("\nInstalling and activating virtual env")
+        create_venv()
+        start_venv()
```

Comment:
so does everything get installed globally for the raspberry pi now?

Reply:
Yes. Since we need numpy pre-compiled, that only comes from apt-get and not pip, but apt-get only installs it on the system and can't be installed in a virtualenv so everything has to be installed globally now.